pages/lessons/projects/3.mdx

```diff
@@ -234,9 +233,9 @@ contract TierNFT is ERC721 {
 }
 ````
 
```

Comment:
I'm only catching on now @7i7o, that you can add comments from within your code editor. Until now, I thought you had to do this in the Github UI. This would have changed a lot, had I known sooner.
Back to the point, I added in the extra couple of lines of contract code so we can explain the mapping a bit more fully in context. My first encounters with mappings wrecked my head! :rofl:

Reply:
I don't see the extra changes in the PR since my review. Have you pushed any more commits to the branch?

Reply:
just did 😸